javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -42,4 +45,20 @@ public PrettyPrinterConfiguration setPrintComments(boolean printComments) {
         this.printComments = printComments;
         return this;
     }
+
+    public PrettyPrinterConfiguration setVisitorClass(Class<? extends PrettyPrintVisitor> visitorClass) {
+        this.visitorClass = visitorClass;
+        return this;
+    }
+
+    public PrettyPrintVisitor createVisitor() {
+        try {
+            Constructor<? extends PrettyPrintVisitor> constructor =
```

Comment:
Please no reflection like this - there is nothing here that requires us to subvert type safety. Why not configure things with a Function<PPC, PPV> ?